src/pages/You.tsx

```diff
@@ -0,0 +1,198 @@
+                let upload = new FormData();
+                upload.append("avatar", images[0]);
+                this.client.patch("/accounts/update_credentials", upload).then((acct: any) => {
+                    let currentAccount: Account = acct.data;
+                    this.setState({ currentAccount });
+                    localStorage.setItem("account", JSON.stringify(currentAccount));
+                    this.props.closeSnackbar("persistAvatar");
+                    this.props.enqueueSnackbar("Avatar updated successfully.");
+                }).catch((err: Error) => {
+                    this.props.closeSnackbar("persistAvatar");
+                    this.props.enqueueSnackbar("Couldn't update avatar: " + err.name, { variant: "error" });
+                })
+            }
+        }).catch((err: Error) => {
+            this.props.enqueueSnackbar("Couldn't update avatar: " + err.name);
+        })
+    }
+
+    updateHeader() {
+        filedialog({
+            multiple: false,
+            accept: "image/*"
+        }).then((images: FileList) => {
+            if (images.length > 0) {
+                this.props.enqueueSnackbar("Updating header...", { persist: true, key: "persistHeader" });
+                let upload = new FormData();
+                upload.append("header", images[0]);
+                this.client.patch("/accounts/update_credentials", upload).then((acct: any) => {
+                    let currentAccount: Account = acct.data;
+                    this.setState({ currentAccount });
+                    localStorage.setItem("account", JSON.stringify(currentAccount));
+                    this.props.closeSnackbar("persistHeader");
+                    this.props.enqueueSnackbar("Header updated successfully.");
+                }).catch((err: Error) => {
+                    this.props.closeSnackbar("persistHeader");
+                    this.props.enqueueSnackbar("Couldn't update header: " + err.name, { variant: "error" });
+                })
+            }
+        }).catch((err: Error) => {
+            this.props.enqueueSnackbar("Couldn't update header: " + err.name);
+        })
+    }
+
+    removeHTMLContent(text: string) {
+        const div = document.createElement('div');
+        div.innerHTML = text;
+        let innerContent = div.textContent || div.innerText || "";
+        return innerContent;
+    }
+
+    //TODO: Implement changeDisplayName
+    changeDisplayName() {
+        this.client.patch('/accounts/update_credentials', { 
+            display_name: this.state.newDisplayName? this.state.newDisplayName: this.state.currentAccount.display_name
+        })
+        .then((acct: any) =>{
+            let currentAccount: Account = acct.data
+            this.setState({currentAccount});
+            localStorage.setItem('account', JSON.stringify(currentAccount));
+            this.props.closeSnackbar("persistHeader");
+            this.props.enqueueSnackbar("Display Name updated successfully.");
```

Comment:
Display Name should be sentence case. Also, I think it'd be nice to also state the new display name in the snackbar so that users don't have to go to their profile to see changes.